Installation/cmake/modules/CGAL_SetupCGAL_CoreDependencies.cmake

```diff
@@ -59,6 +59,14 @@ if (NOT MSVC)
   find_package( Boost 1.48 REQUIRED COMPONENTS thread system )
 endif()
 
+function(CGAL_Core_create_alias_Exact_rational_support)
+  if(NOT TARGET CGAL::Exact_rational_support)
```

Comment:
I have reverted the commit.

But anyway, the name should have been `Exact_algebraic_support`.